mobile/host.go

```diff
@@ -106,10 +106,17 @@ func create(ctx context.Context, configRoot string) (*rhost.RoutedHost, error) {
 
 	// Make the DHT
 	kDht := dht.NewDHT(ctx, basicHost, dstore)
+	cfg.Bootstrap = append(cfg.Bootstrap,
+		"/ip4/34.224.40.105/udp/4001/quic/p2p/12D3KooWEftKAarKSc1bhQfgn5aoW5UnaSqCr9UMhRoqhsBA6MmX",
+		"/ip4/54.235.11.104/udp/4001/quic/p2p/12D3KooWEHmZunko2dupAR9J3Ydo3yN8aW7oZWkAxv5zsNL7UPRH",
```

Comment:
I am curious where these addresses are coming from?

Reply:
This is my doing these are node run on AWS as bootstraps.
There is problem with resolvin dnsaddr multiaddress in android so I end up creating this two node so there is no need for resolve dnsaddr.

Reply:
Do we need to check these to code as a permanent change or is it meant to be temporary? 

If former, I recommend adding inline comments to explain what the multiaddrs point to.

Reply:
```suggestion
                // This is a temrary fix for issue 43 (also ipfs suggest to have your own bootstrap node), this multi addresses point to Functionland own bootstrap nodes.
                // TODO: change ip4 to dns4 addresses
		"/ip4/54.235.11.104/udp/4001/quic/p2p/12D3KooWEHmZunko2dupAR9J3Ydo3yN8aW7oZWkAxv5zsNL7UPRH",
```

Reply:
Something like this?
```
                // This is a temrary fix for issue 43 (also ipfs suggest to have your own bootstrap node), this multi addresses point to Functionland own bootstrap nodes.
                // TODO: change ip4 to dns4 addresses
```

Reply:
@mehdibalouchi can you add it?